app/assets/stylesheets/_variables.scss

```diff
@@ -26,29 +21,14 @@ $dark-gray: #333;
 $medium-gray: #999;
 $light-gray: #DDD;
 
-// Background Color
-$base-background-color: white;
```

Comment:
Alright. It can be added on an as-needed basis. I don't think it should be with the button; it's not button-specific.